apps/angular/src/core/services/anime.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { HttpParams } from '@angular/common/http';
+import { map, Observable } from 'rxjs';
+
+import { Pagination } from '@js-camp/core/models/pagination';
+import { AnimeDto } from '@js-camp/core/dtos/anime.dto';
+import { PaginationDto } from '@js-camp/core/dtos/pagination.dto';
+import { AnimeMapper } from '@js-camp/core/mappers/anime.mapper';
+import { PaginationMapper } from '@js-camp/core/mappers/pagination.mapper';
+import { Anime } from '@js-camp/core/models/anime';
+
+import { ApiService } from './api.service';
+
+const DEFAULT_SEARCH_OPTIONS = new HttpParams({
+  fromString: 'limit=10&offset=0&ordering=id',
+});
```

Comment:
Prefer using `fromObject` instead:
```suggestion
const DEFAULT_SEARCH_OPTIONS = new HttpParams({
  fromObject: {
    limit: 10,
    offset: 0,
    ordering: 'id'
  }
});
```
Maybe it's more verbose in this case, but in general it is more convenient to work with objects, for example it allows you to use the spread operator. Also you don't have to manually concatenate params into one string